service/history/nDCStateRebuilder.go

```diff
@@ -191,7 +200,7 @@ func (r *nDCStateRebuilderImpl) rebuild(
 func (r *nDCStateRebuilderImpl) initializeBuilders(
 	namespaceEntry *namespace.Namespace,
 	now time.Time,
-) (workflow.MutableState, workflow.MutableStateRebuilder) {
+) (*workflow.MutableStateImpl, workflow.MutableStateRebuilder) {
```

Comment:
why this change? is caller now depends on the implementation detail of MutableStateImpl?